exposed/CsDemo/.gitignore

```diff
@@ -0,0 +1,369 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Adding such a huge .gitignore seems like bad form to me. One that ignores only the specific files and directories created by VS when building the example is (in my opinion) better. Currently I do not have the C# build tools in my VS installation but I will add them (when I get to my development system) and try the example.

Reply:
It is a 'standard' .gitignore from [here](https://github.com/github/gitignore/blob/main/VisualStudio.gitignore)

Reply:
That is clear from the comment at the top.